corehq/apps/data_dictionary/tests/test_util.py

```diff
@@ -210,15 +210,21 @@ def test_get_column_headings(self):
         self.assertEqual(errors, expected_errors)
 
     def test_map_row_values_to_column_names(self):
-        column_headings, _ = get_column_headings(self.valid_header_row, self.valid_values)
-        row_vals = map_row_values_to_column_names(self.row, column_headings, default_val='empty')
+        sheet_name = 'foobar'
+        column_headings, _ = get_column_headings(self.valid_header_row, self.valid_values, sheet_name)
+        row_vals, errors = map_row_values_to_column_names(
+            self.row, column_headings, sheet_name, default_val='empty'
+        )
         expected_output = {
             'col_1': 'a',
             'col_2': 'b',
             'col_3': 'empty',
         }
+        self.assertEqual(len(errors), 0)
         for key, val in expected_output.items():
             self.assertEqual(row_vals[key], val)
+        row_vals, errors = map_row_values_to_column_names(self.row, column_headings[0:1], sheet_name=sheet_name)
```

Comment:
nit: consider putting this test in a separate test method for better isolation of the failure scenario.

Reply:
[9bdcafa](https://github.com/dimagi/commcare-hq/pull/34368/commits/9bdcafacbfc4e5fa9f8b908d721d6b0a3426d287)